package.json

```
@@ -65,7 +65,7 @@
     "build": " gatsby build --prefix-paths",
     "develop": "npm run clean && gatsby develop",
     "format": "prettier --trailing-comma es5 --no-semi --single-quote --write \"{gatsby-*.js,src/**/*.js,src/**/*.json}\"",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "true"
```

I much prefer to still log an error so that it can be seen in logs as necessary, eg
```suggestion
    "test": "echo \"Error: no test specified\""
```